Cargo.toml

```diff
@@ -19,3 +19,7 @@ url = "1.2"
 
 [dev-dependencies]
 env_logger = "0.3"
+
+[[example]]
+name = "simple"
+path = "examples/simple.rs"
```

Comment:
Oh, I was not aware of that. Thanks for letting me know. I remove this part of the PR this evening.